argilla-server/tests/unit/api/handlers/v1/datasets/fields/test_list_dataset_fields.py

```diff
@@ -19,7 +19,7 @@
 
 from argilla_server.enums import FieldType
 
-from tests.factories import DatasetFactory, ImageFieldFactory
+from tests.factories import DatasetFactory, ImageFieldFactory, ChatFieldFactory
```

Comment:
Unrelated, but why don't we have tests for TextFields here?

Reply:
I'm not sure why they're not here. `TextFieldFactory` is tested in `test_datasets.py`.